### PR TITLE
add install log regexp for route53 timeout

### PR DESCRIPTION
**Pull request by gregsheremeta**
/assign @joelddiaz

### CODE REVIEW

**Comment by joelddiaz** [comment]
I thought we decided this was an underlying throttling issue hidden by terraform? https://github.com/openshift/hive/pull/1629#issuecomment-990998364

**Comment by gregsheremeta** [comment]
>  thought we decided this was an underlying throttling issue hidden by terraform?

indeed. Fixed, ptal

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 1]
```diff
@@ -88,6 +88,14 @@ data:
       - "Throttling: Rate exceeded"
       installFailingReason: AWSAPIRateLimitExceeded
       installFailingMessage: AWS API rate limit exceeded
+    # This issue is caused by AWS throttling the CreateHostedZone request. The terraform provider is not properly
+    # handling the throttling response and gets stuck in a state where it does not retry the request. Eventually,
+    # the terraform provider times out claiming that it is waiting for the hosted zone to be INSYNC.
+      - name: AWSRoute53Timeout
```
Comment (2uasimojo) [inline config/configmaps/install-log-regexes-configmap.yaml:94]: Unindent this. And don't forget to rerun `make update`.

Comment (gregsheremeta) [reply config/configmaps/install-log-regexes-configmap.yaml]: yikes. done.

**Comment by 2uasimojo** [comment]
/lgtm

So is this one going to be retryable?

**Comment by gregsheremeta** [comment]
> So is this one going to be retryable?

yes, sent the MR already

**Pull request merged**